admin/data/sql/migrations/20181129141323.undo.add-islivecheckform-to-checkform.sql

```diff
@@ -0,0 +1,13 @@
+DECLARE @SQL varchar(2000);
+DECLARE @CONSTRAINT_NAME varchar(2000);
+
+SET @CONSTRAINT_NAME = (SELECT name from SYS.DEFAULT_CONSTRAINTS
+                        WHERE PARENT_OBJECT_ID = OBJECT_ID('mtc_admin.checkForm')
+                          AND PARENT_COLUMN_ID = (SELECT column_id FROM sys.columns
+                                                  WHERE NAME = N'isLiveCheckForm'
+                                                    AND object_id = OBJECT_ID('mtc_admin.checkForm')))
+
+SET @SQL = 'ALTER TABLE [mtc_admin].[checkForm] DROP CONSTRAINT ' + @CONSTRAINT_NAME
```

Comment:
did you try just dropping the column?  the default should be dropped with it.
Or, if not, use a named constraint and simplify to `ALTER TABLE [mtc_admin].[checkForm] DROP CONSTRAINT isLiveCheckFormDefault`